cmd/cmd.go

```diff
@@ -140,7 +140,8 @@ func InternalToolPath(tool string) string {
 	return filepath.Join(filepath.Dir(exe), tool)
 }
 
-// mustUnsetenv will os.Unsetenv the for or panic if it cannot do that
+// mustUnsetenv will unset the given environment key or panic if it
+// cannot do that
```

Comment:
s/will unset/unsets/, if we are to follow the traditional style of Go comments for functions and methods.